Code/GraphMol/Descriptors/testATF.cpp

```diff
@@ -0,0 +1,59 @@
+//  Created by Guillaume GODIN
+//  Copyright (C) 2012-2017 Greg Landrum
+//   @@ All Rights Reserved @@
+//
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+#include <RDGeneral/test.h>
+#include <RDGeneral/Invariant.h>
+#include <GraphMol/RDKitBase.h>
+#include <GraphMol/GraphMol.h>
+#include <GraphMol/FileParsers/MolSupplier.h>
+#include <GraphMol/FileParsers/MolWriters.h>
+#include <GraphMol/SmilesParse/SmilesParse.h>
+#include <RDGeneral/RDLog.h>
+#include <vector>
+#include <algorithm>
+#include <fstream>
+#include <sstream>
+
+#include <chrono>  // for high_resolution_clock
+
+#include <GraphMol/Descriptors/AtomFeat.h>
+
+void test1() {
```

Comment:
I'd like to see a bit more testing here. You only check a single atom type. Please pick a molecule that has different types of atoms (in terms of degree, hybridization, charge, and ring info) and check a few different atoms.
Perhaps this one: `C1CC=C1[NH3+]`
Also add a separate test with two chiral atoms to make sure that the chiral flag works (i.e. look at vectors for both atoms when `addchiral` is `true` and `false`)
Here's a good molecule for that: `O[C@H](F)[C@H](F)O`